qa-acct.planx-pla.net/manifest.json

```diff
@@ -0,0 +1,69 @@
+{
+  "notes": [
+    "This is the account-qa environment manifest",
+    "That's all I have to say"
+  ],
+  "jenkins": {
+    "autodeploy": "yes"
+  },
+  "versions": {
+    "arborist": "quay.io/cdis/arborist:2.3.2",
+    "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
+    "fence": "quay.io/cdis/fence:4.13.2",
+    "indexd": "quay.io/cdis/indexd:2.5.0",
```

Comment:
Maybe we should bump that up too? 😄 
Latest release version: `indexd:2.6.1`

Reply:
:facepalm: Thanks